slemicro/combustion/04_elemental_register.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -euo pipefail
+
+if [ "${ELEMENTAL_REGISTER}" = true ]; then
+  # Install elemental-register and elemental-system-agent
+  # Dev version currently needed for new --no-toolkit option
+  zypper ar --refresh --no-gpgcheck https://download.opensuse.org/repositories/isv:/Rancher:/Elemental:/Dev/standard/isv:Rancher:Elemental:Dev.repo
+  zypper -n in elemental-register elemental-system-agent
+  mkdir -p /etc/elemental
+  cp ./elemental_config.yaml /etc/elemental/config.yaml
+
+  # Register --no-toolkit disables OS management
+  elemental-register --config-path /etc/elemental/config.yaml --state-path /etc/elemental/state.yaml --install --no-toolkit --emulate-tpm --emulated-tpm-seed 99
+
```

Comment:
Note that the emulated TPM is required currently - I tried to get the TPM support that landed in the latest UTM versions working, but SLEMicro won't boot - I'll try to figure that out and we can potentially resolve in a follow-up

Reply:
I'm assuming that the TPM issue is only for UTM and not bare-metal? (I don't think I had this issue with my NUCs)